_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title:  "Trino 2021 Wrapped: A Year of Growth"
+author: Brian Olsen
```

Comment:
Probably should have at least Martin added as coauthor

Reply:
Added you both